workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/CodeGenerator.cs

```diff
@@ -104,6 +109,12 @@ private HashSet<string> ExtractEnums(ICollection<UnitySchemaFile> schemas)
             return enumSet;
         }
 
+        private List<string> ExtractWorkerTypes(string path)
+        {
+            var fileNames = Directory.EnumerateFiles(path, "spatialos.*.worker.json");
+            return fileNames.Select(fileName => Path.GetFileName(fileName).Split('.')[1]).ToList();
```

Comment:
How about moving the `GetFileName` call into a separate `.Select(Path.GetFileName).Select(fileName => fileName.Split('.')[1])`